### PR TITLE
Feat/threats to amazon

### PR DESCRIPTION
**Pull request by Ali-Aizaz**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -183,6 +184,9 @@ export default {
       3: 3,
       4: 4,
     },
+    maxWidth: {
+      46: "11.5rem",
```
Comment (Kokanm) [inline tailwind.config.ts:188]: What was the reason for this addition? Is it that size in Figma or it makes things look better?

File: src/hooks/useGetAboutTheAmazonContent.tsx [hunk 2]
```diff
@@ -1082,6 +1095,79 @@ export function useGetAboutTheAmazonContent() {
         },
       },
     },
+    {
+      type: "regular",
```
Comment (Kokanm) [inline src/hooks/useGetAboutTheAmazonContent.tsx:1099]: Will you add the data from Sanity in a different PR?

**Review by Kokanm — APPROVED** [review]

File: src/hooks/useGetAboutTheAmazonContent.tsx (hunk 2 above)
Comment (Ali-Aizaz) [reply src/hooks/useGetAboutTheAmazonContent.tsx:1099]: Yes

Comment (Kokanm) [reply src/hooks/useGetAboutTheAmazonContent.tsx:1099]: No need, you can add it in a different PR

**Pull request merged**